logging/tests/system/test_system.py

```diff
@@ -16,14 +16,11 @@
 import logging
```

Comment:
No package initializer?  And why move it into a subdir?

Reply:
Not sure what you mean by package initializer. Moved to subdir because gapic has its own system tests.

Reply:
Package initializer: `tests/system/__init__.py`.
py.test does not require it as long as you do not try to reuse the same filename.